src/DynamoCoreWpf/Properties/Resources.Designer.cs

```diff
@@ -747,6 +747,15 @@ public class Resources {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to Crash report from Dynamo {0}.
+        /// </summary>
+        public static string CrashPromptGithubNewIssueTitle {
+            get {
+                return ResourceManager.GetString("CrashPromptGithubNewIssueTitle", resourceCulture);
+            }
+        }
```

Comment:
Brackets should follow the [coding standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards#conditional-blocks), such as
```C#
public static string CrashPromptGithubNewIssueTitle 
{
    get 
    {
        return ResourceManager.GetString("CrashPromptGithubNewIssueTitle", resourceCulture);
    }
}
```

I do this all the time accidentally switching between C# and Javascript

Reply:
that's auto-generated code from Visual Studio (to handle `Resources`) and will be overwritten if I modify anyway - unless i'm missing something ?

Reply:
From the comments at the top of the file I believe that this is correct, but if it is auto-generated then it should not be a part of the PR.

Reply:
It’s in repo already https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/Properties/Resources.Designer.cs

File is part of how VisualStudio stores string resources, not something I added

Reply:
I'm not sure if it matters if it is in the repo or not, just that it will be recreated when the build is run so it is probably not necessary.

Reply:
Yeah, this is going to be regenerated all the time. The format in `Resources.Designer.cs` should be fine as long as it follows the other resource strings.